src/app/util/decode_webp.h

```diff
@@ -0,0 +1,31 @@
+// Aseprite
+// Copyright (C) 2024  Igara Studio S.A.
+//
+// This program is distributed under the terms of
+// the End-User License Agreement for Aseprite.
+
+#ifndef APP_UTIL_DECODE_WEBP_H_INCLUDED
+#define APP_UTIL_DECODE_WEBP_H_INCLUDED
+#pragma once
+
+#include "os/surface.h"
+#include "os/system.h"
+
+namespace app {
+
+namespace util {
+
+// Decodes webp content passed in buf and returns a surface with just the first
+// frame.
+os::SurfaceRef decode_webp(const uint8_t* buf, uint32_t len);
+
+static void* once = []() {
+  os::set_decode_webp(decode_webp);
```

Comment:
The `os::set_decode_webp()` function is missing, probably a missing `laf` submodule update. Anyway we cannot initialize variables like this, this code should be moved inside a function.

Reply:
I thought about adding the default decoding code in laf, but I would need to add a dependency to libwebp library. If that's ok I can do it.

> Anyway we cannot initialize variables like this, this code should be moved inside a function.

Sure, in which function do you think it would be the best place to put this?

Reply:
In any place you can think of that will make sense, but not as a global function call outside function bodies. This kind of initialization is like calling a constructor/a global variable initialization, and C++ doesn't ensure a proper initialization order between different compilation units.